### PR TITLE
[docs-only] Review text in sample config PHP files

### PR DESCRIPTION
**Pull request by phil-davis**
## Description
I noticed some uses of "he" that should not be gender-specific. I fixed those, and various other things that the IDE found/suggested.

There is nothing exciting here that needs to be mentioned in the changelog.

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests only (no source changes)

## Checklist:
- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 
- [ ] Changelog item, see [TEMPLATE](https://github.com/owncloud/core/blob/master/changelog/TEMPLATE)

### CODE REVIEW

**Comment by phil-davis** [comment]
When this is transferred across to docs, we can decide if we care to backport any of the changes to the 10.8 and 10.7 docs.

File: config/config.sample.php [hunk 1]
```diff
@@ -1659,10 +1659,10 @@
 
 /**
  * Link to get a demo key during active grace period
- * The admin will be directed to that web page if he clicks in the "get a demo key"
+ * The admin will be directed to that web page if they click in the "get a demo key"
```
Comment (JanAckermann) [inline config/config.sample.php:1662]:
```suggestion
 * The admin will be directed to that web page while clicking in the "get a demo key"
```

**Review by AlexAndBear — APPROVED** [review]
LGTM
hyphenated form might be a little to much, but that is just my POV, as I don't like em anyways

**Comment by phil-davis** [comment]
> hyphenated form might be a little to much, but that is just my POV, as I don't like em anyways

The grammar-checker in PHPstorm "shuts" at me to add the hyphens.

**Pull request merged**